components/token/LockedBANKSection.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import Image from 'next/image'
+
+import {
+  Flex,
+  Text,
+  Container,
+  VStack,
+  Box,
+  Link,
+  useMediaQuery,
+} from '@chakra-ui/react'
+
+function LockedBANKSection() {
+  const [isMobile] = useMediaQuery('(max-width: 768px)')
+
+  return (
+    <Container maxW={'6xl'} p={0} pt='60px'>
+      <Flex direction={{ base: 'column', xl: 'row' }}>
+        <VStack spacing={4} align='flex-start'>
+          <Text fontWeight={700} fontSize='7xl'>
+            Time Locked BANK
+          </Text>
+          <Text fontSize='3xl' fontWeight={700} color={'#D02128'}>
+            Long-term alignment{' '}
+          </Text>
+          <Text fontWeight={400} lineHeight='22px' fontSize='lg'>
+            Locking up BANK creates Time Locked BANK (tlBANK) and signals
+            long-term alignment with BanklessDAO and it&apos;s mission. When you
+            Lock BANK you receive a non-transferable tlBANK NFT that grants
+            Level 1 membership status for the duration of the lock period. You
+            also receive transferable tlBANK tokens, equivalent to the amount of
+            BANK locked. Only the address that mints the tlBANK is entitled to
+            the Level 1 membership status and other future perks.
```

Comment:
@Icedcool Please pay special attention to the tlBank copy, here.

Reply:
Making a few edits to this.

Reply:
Need to change to:
`Locking up BANK creates Time Locked BANK (tlBANK) and signals long-term alignment with BanklessDAO and it&apos;s mission. When you Lock BANK you receive a tlBANK NFT that represents the amount of BANK locked and grants Level 1 membership status for the duration of the lock period.  Only the address that mints the tlBANK is entitled to the Level 1 membership status and other future perks.`
Or similar such edit.